test/vagrant-service-manager/installer_test.rb

```diff
@@ -29,7 +29,7 @@ module ServiceManager
           @installer.instance_variable_get('@type').must_equal @type
           @installer.instance_variable_get('@machine').must_equal @machine
           @installer.instance_variable_get('@env').must_equal @machine.env
-          @installer.instance_variable_get('@box_version').must_equal 'adb'
+          @installer.instance_variable_get('@options').must_equal({ type: @type }.merge(@options))
```

Comment:
@hferentschik, Ya agree. Just checking whether we are getting expected variables or not.